src/storage/icons.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useEffect } from 'react';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+
+type Icons = string[];
+type ModifyIcons = {
+  add: (p: string) => boolean;
+  swap: (a: number, b: number) => boolean;
+};
+
+export const iconsKey: string = "icons";
+export const iconsDefault: Icons = [
+  "folder",
+  "person",
+];
+
+/**
+ * Hook returning a list of icons and functions to modify the list.
+ *
+ * @example
+ * Get stored icons and print them:
+ * ```
+ * const [icons, modifyIcons] = Storage.useIcons();
+ * console.log(icons);
+ * ```
+ *
+ * @example
+ * Swap two icons in the list:
+ * ```
+ * const [icons, modifyIcons] = Storage.useIcons();
+ * modifyIcons.swap(0, 1);
+ * ```
+ */
+export const useIcons = (): [Icons, ModifyIcons] => {
+  const [icons, setIcons] = useState<Icons>([]);
+
+  /**
+   * Adds an icon to the list of icons and updates AsyncStorage.
```

Comment:
Is there a specific icon form required? Eg. JPG? 
If so, it maybe should be described here?